pom.xml

```diff
@@ -158,6 +158,8 @@
 
     <modules>
         <module>bom/public</module>
+        <module>bom/platform</module>
+        <module>bom/platform-next</module>
```

Comment:
Maven 4 wording would be `bom/skinny` vs. `bom/fat`, or something like that. Though I'm sure someone will get offended at the use of the word "fat" :roll_eyes: 

Did you look into conventions before going with the `platform` name? And yes, I know that's what Hibernate ORM uses.